tox.ini

```
@@ -60,10 +60,10 @@ commands =
     coverage run {env:COVERAGE_OPTIONS:} {envbindir}/django-admin test -v 2 --pythonpath=./ --settings=tests.settings
     coverage report
 
-[testenv:flake8]
+[testenv:ruff]
 basepython = python3
 extras = linting
-commands = flake8 example tests two_factor
+commands = ruff
```

I think this will fix the failing github action:

```suggestion
commands = ruff .
```